docs/reference/contributing/guidelines/workflow.md

```diff
@@ -51,6 +51,30 @@ Pull requests on GitHub have to meet the following requirements to keep the code
 - Prefixing your commit message with a domain is acceptable, and we recommend doing so when it makes sense. However, prefixing one's domain with the name of the repo is not useful. For example, making a commit entitled "mbed-drivers: Fix doppelwidget frobulation" to the `mbed-drivers` repo is not acceptable because it is already understood that the commit applies to `mbed-drivers`. Renaming the commit to "doppelwidget: Fix frobulation" would be better, if we presume that "doppelwidget" is a meaningful domain for changes, because it communicates that the change applies to the doppelwidget area of `mbed-drivers`.
 - All new features and enhancements require documentation, tests and user guides for us to accept them. Please link each pull request to all relevant documentation and testing pull requests.
 
+##### Pull request template
+
+A good example how a pull request template should look like.
+
+```
+Fix deep sleep locking bug
+
+# Description
+
+Fix problems which could leave deep sleep locked unintentionally, along with adding tests to verify this behavior is fixed.
+
+Tested locally with two targets, all toolchains.
```

Comment:
I would recommend we be specific here, to encourage others to be specific as well.

Replace "two targets" with something like "K64F and K66F".